core/src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/CompactEntityComponentSpec.scala

```diff
@@ -40,6 +40,9 @@ class CompactEntityComponentSpec extends TestDriverComponentWithFlatSpecAndMatch
   private val ws2id = minimalTestData.workspace2.workspaceIdAsUUID
   private val q = compactEntityQuery
 
+  // for raw sql queries
+  import slickDataSource.dataAccess.compactEntityQuery.{GetUUIDResult, SetUUIDParameter}
+
```

Comment:
this also must have unintentionally relied on imports/implicits from the bucket migration queries or test code